test/Driver/batch_mode_response_files.swift

```diff
@@ -0,0 +1,10 @@
+// Batch jobs go through a different code path than other jobs, so make sure
```

Comment:
It may also be worth updating the test for -driver-force-response-files to ensure that it also works for batch mode.

Reply:
👍  Added.